lib/src/migrators/module/unreferencable_members.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+// The sass package's API is not necessarily stable. It is being imported with
+// the Sass team's explicit knowledge and approval. See
+// https://github.com/sass/dart-sass/issues/236.
+import 'package:sass/src/ast/sass.dart';
+
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p;
+
+import 'package:sass_migrator/src/utils.dart';
+import 'unreferencable_type.dart';
+
+/// Tracks members that are unreferencable in the current scope.
+class UnreferencableMembers {
```

Comment:
I really like making this explicitly track exactly the set of members that are unreferencable. Given that we need special handling for these (which I believe), making them super explicit keeps the code readable and the responsibilities clear.